ch01.asciidoc

```diff
@@ -62,7 +71,11 @@ Therefore, Bob can securely receive more updates to the channel balance as long
 
 This mechanism would allow two users to engage in several smaller transactions which all happened outside of the Bitcoin network.
 While this construction of the unidirectional payment channel would have solved the custody problem of exchanges it has never been widely implemented.
-We can only speculate for reasons and guess that the overhead communication would have had to be standardized - as it is nowadays in the LN specification - which might have been too much overhead in the early days of Bitcoin.
+
+A fundamental blocker to deploying payment channels in production was the issue of transaction malleability. 
```

Comment:
I'm thinking in the series above (attempt at secure channels by construction), we can introduce malleability as the final barrier after being able to enforce commitment revocation.